lib/models/os.ts

```diff
@@ -957,6 +957,32 @@ const getOsModel = function (
 		);
 	};
 
+	/**
+	 * @summary Returns image name for a specific supervisor version for an OS architecture
+	 * @name getSupervisorImageForArch
+	 * @public
+	 * @function
+	 * @memberof balena.models.os
+	 *
+	 * @param {String} architecture - The OS's architecture as specified in its device type
+	 * @param {String} version - The semver version string for the supervisor
+	 * @returns {Promise<String>} - Docker image name for the Supervisor version and arch
+	 *
+	 * @example
+	 * const result = balena.models.os.getSupervisorImageForArch('aarch64', 'v12.11.0').then(result => console.log(result))
+	 *
+	 */
+	const getSupervisorImageForArch = async (
+		architecture: string,
+		version: string,
+	): Promise<string | null> => {
+		const { body } = await request.send({
+			url: `/v6/supervisor_release?\$top=1&\$select=image_name&\$filter=(supervisor_version%20eq%20%27${version}%27)%20and%20(is_for__device_type/any(ifdt:ifdt/is_of__cpu_architecture/any(ioca:ioca/slug%20eq%20%27${architecture}%27)))`,
+			baseUrl: apiUrl,
+		});
```

Comment:
Reminder to change this to a proper `pine.get()`.

Reply:
done